src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -298,7 +298,7 @@ function RetroCertsCertificationPage(props) {
                 inputName="recentDisaster"
               >
                 <DisasterQuestion
-                  questionText={t(questionText("recentDisasterChoice"))}
+                  questionText={t(questionText("disasterChoice"))}
```

Comment:
I'd rather make everything `recentDisasterChoice` to be consistent with `recentDisaster`, but all the form entries in the db already use `disasterChoice` so i'm standardizing on that